distribution/docs/src/main/resources/_contents/_endpoints/kml-endpoint-contents.adoc

```diff
@@ -0,0 +1,80 @@
+The query parameters for this query are obtained by a bounding box generated by Google Earth.
+The root network link will refresh every 12 hours or can be forced to refresh.
+As a user changes their current view, the query will be re-executed with the bounding box of the new view.
+(This query gets re-executed two seconds after the user stops moving the view.)
+
+===== Installing the KML Endpoint
+
+The KML Network Link Endpoint is installed by default with a standard installation as part of the ${ddf-spatial} application.
+
+===== Configuring the KML Endpoint
+
+This KML Network Link endpoint has the ability to serve up custom KML style documents and Icons to be used within that document.
+The KML style document must be a valid XML document containing a KML style.
+The KML Icons should be placed in a single level directory and must be an image type (png, jpg, tif, etc.).
+The Description will be displayed as a pop-up from the root network link on Google Earth.
+This may contain the general purpose of the network and URLs to external resources.￼￼￼
+
+include::{adoc-include}/_tables/org.codice.ddf.spatial.kml.endpoint.KmlEndpoint-table-contents.adoc[]
+
+===== Using the KML Endpoint
+
+Once installed, the KML Network Link endpoint can be accessed at:
+----
+${secure_url}/services/catalog/kml
+----
+
+After the above request is sent, a KML Network Link document is returned as a response to download or open.
+This KML Network Link can then be opened in Google Earth.
+
+.Example Output from KML Endpoint
+[source,xml,linenums]
+----
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<kml xmlns="http://www.opengis.net/kml/2.2" xmlns:ns2="http://www.google.com/kml/ext/2.2"
+  xmlns:ns3="http://www.w3.org/2005/Atom" xmlns:ns4="urn:oasis:names:tc:ciq:xsdschema:xAL:2.0">
+  <NetworkLink>
+    <name>${branding}</name>
+    <open xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xs="http://www.w3.org/2001/XMLSchema" xsi:type="xs:boolean">true</open>
+    <Snippet maxLines="0"/>
+    <Link>
+      <href>http://0.0.0.0:8181/services/catalog/kml/sources</href>
+      <refreshMode>onInterval</refreshMode>
+      <refreshInterval>43200.0</refreshInterval>
+      <viewRefreshMode>never</viewRefreshMode>
+      <viewRefreshTime>0.0</viewRefreshTime>
+      <viewBoundScale>0.0</viewBoundScale>
+    </Link>
+  </NetworkLink>
+</kml>
+----
+
+When configured to do so, the KML endpoint can serve up a KML style document.
+The request below will return the configured KML style document.
+
+.KML-style Document
+----
+${secure_url}/services/catalog/kml/style
+----
+
+The KML endpoint can also serve up Icons to be used in conjunction with the KML style document.
+The request below shows the format to return an icon.
+
+.Return KML Icon
+----
+${secure_url}/services/catalog/kml/icons?<icon-name>
+#NOTE: <icon-name> must be the name of an icon contained in the directory being served
```

Comment:
The _#NOTE:_ should probably be outside of the block, otherwise it looks bad in PDF.